docs/changes/newsfragments/3688.improved_driver

```diff
@@ -0,0 +1,4 @@
+Update N52xx driver to fix warnings on initialization
+related to not passing **kwargs in trace parameters (closes #3687).
```

Comment:
@spauka This should be valid sphinx rst. In rst `**` starts a bold section (Inline strong) this never ends here. I suggest that we wrap this in double ticks ``` ``**kwargs`` ```

Reply:
Thanks for the tip - that seems to have fixed it :)